docs/modules/extend/pages/use-cases.adoc

```diff
@@ -57,6 +57,46 @@ The method `stroke_horizontal_rule` draws a horizontal line using the specified
 Finally, the method `convert` will convert and render the Asciidoctor node that is passed to it, in this case a block image.
 
+== Custom document header without a title page
+
+Documents rendered as articles by Asciidoctor PDF don't have a title page by default.
+Instead their document title is shown on the first content page above the actual content.
+
+Other than in articles rendered by Asciidoctor to HTML, and other than when rendered with a title page (e.g. document type book),
+the author, date and revision remark won't be shown by default.
+To get them included anyway, we can hook into the method `ink_general_heading`, which renders the document title.
+As `ink_general_heading` is also responsible for rendering other "headers" (e.g. section titles),
+we let it fall back to the default implementation for all headers that aren't the document title.
+
+.Extended converter with author, date and revision remark below the document title of document types without a title page
+[,ruby]
+----
+include::example$pdf-converter-article-title-with-author-and-date.rb[]
+----
+<.> Fall back to default implementation unless handling the document title.
+<.> Render the document title.
+See the definition of `ink_document_title` for details.
+<.> Render author, date and revision remark, if they're available.
+See the definition of `ink_document_metainfo` for details.
+<.> Render a vertical gap between the title section (incl. author, date and revision remark) and the body of the document.
+See the definition of `ink_titlesection_body_separator` for details.
+<.> The revision remark isn't available as a field of `doc`, so we have to get it from the hash `doc.attributes`.
+We could do that with `doc.attributes['revremark']`, but we use the convenience method `doc.attr` instead.
+Because we'll use the revision remark multiple times, we store it in a local variable.
+<.> Only include the additional line if there's actually content for it, so we don't end up with unused vertical space.
+<.> Render a vertical gap between the document title
+and the line with author, date and revision remark
+if the theme sets a size of that gap.
+Otherwise, fall back to a zero-height "`gap`".
+<.> When we get here, a heading-specific style will already be set.
+(This is ensured by the methods that call `ink_general_heading`.)
+As we don't want author, date and revision remark to look like a heading,
+we first have to reset to the default style using the `:base` category.
+Then, we can apply the `heading_h1_details` category.
+<.> An em-dash to be put between author and date, but only if both aren't empty.
+<.> A vertical bar to be put before the revision remark, but only if it isn't empty and there's actually something else (author, date or both) in front of it.
+<.> Put author, date, revision remark and the separators together using Ruby https://docs.ruby-lang.org/en/3.0/syntax/literals_rdoc.html#label-Strings[string] interpolation and render the resulting string.
```

Comment:
I don't think the link to the Ruby docs is necessary here. First, it's linking to a specific version, which will fall out of date and I will have to update. But I also think that it's up to the reader to find references to the Ruby documentation, because we could be adding links all over the place.